src/frontend/apps/e2e/__tests__/app-desk/teams-create.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { expect, test } from '@playwright/test';
+
+import { keyCloakSignIn } from './common';
+
+test.beforeEach(async ({ page, browserName }) => {
+  await page.goto('/');
+  await keyCloakSignIn(page, browserName);
+});
+
+test.describe('Teams', () => {
+  test('checks all the create box elements are visible', async ({ page }) => {
+    const buttonCreateHomepage = page.getByRole('button', {
+      name: 'Create a new team',
+    });
+    await buttonCreateHomepage.click();
+    await expect(buttonCreateHomepage).toBeHidden();
+
+    const panel = page.getByLabel('Panel create new team').first();
```

Comment:
Again, naming both the central card and the left panel with the same name could be confusing !

Reply:
[30e0c2f](https://github.com/numerique-gouv/people/pull/43/commits/30e0c2fb85973c4fa090ec739429b1535193d1a2)